clusters/build-clusters/02_cluster/assets/99-crun-control-plane-machineconfig.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: control-plane
+  name: 99-control-plane-crun
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lID0gImNydW4iCltjcmlvLnJ1bnRpbWUucnVudGltZXMuY3J1bl0K
```

Comment:
Just for the record:
```console
$ echo -n "W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lID0gImNydW4iCltjcmlvLnJ1bnRpbWUucnVudGltZXMuY3J1bl0K" |base64 -d
[crio.runtime]
default_runtime = "crun"
[crio.runtime.runtimes.crun]
```

Reply:
Oh. Just saw https://github.com/openshift/release/pull/31199#issuecomment-1209782307